api/grpc_server.go

```diff
@@ -385,7 +385,7 @@ func (s SpacemeshGrpcService) SetLoggerLevel(ctx context.Context, msg *pb.SetLog
 }
 
 func (s SpacemeshGrpcService) GetAccountTxs(ctx context.Context, txsSinceLayer *pb.GetTxsSinceLayer) (*pb.AccountTxs, error) {
-	log.Info("GRPC GetAccountTxs msg")
```

Comment:
I wouldn't change the print level since it helps understand if a rpc was made or not and doesn't pollute logs too much

Reply:
@barakshani says it does pulute (he asked me to silence them)